module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2029,7 +2056,11 @@ protected function getItemStatusesForBiblio($id, $patron = null)
             return [];
         }
 
-        $statuses = [];
+        // Get the total number of items returned from the API call and save it to
+        // a local array. It is then used in VuFind\RecordTab\HoldingsILS for
+        // the items paginator
+        $results['total'] = (int)$result['data']['items_total'];
```

Comment:
I'd avoid documenting functionality elsewhere in too much detail, because it could change later and cause confusion, so I suppose something like "Return total number of results for pagination." would be plenty enough here, but I won't hold this any longer because of that. :)

Reply:
I've gone ahead and made the suggested edit prior to merging this.